docs/function-format.md

```diff
@@ -58,42 +62,44 @@ The header keys and values would be populated with information about the functio
 
 Pros:
 
-* Streamable
+* Supports streaming
 * Common format
 
 Cons:
 
 * Requires a parsing library or fair amount of code to parse headers properly
 * Double parsing - headers + body (if body is to be parsed, such as json)
 
-#### JSON I/O Format (not implemented)
+#### JSON Input Format
 
 `--format json`
 
-The idea here is to keep the HTTP base structure, but make it a bit easier to parse by making the `request line` and `headers` a JSON struct.
-Eg:
+An easy to parse JSON structure.
 
-```
+```json
 {
-  "request_url":"http://....",
-  "params": {
-    "blog_name": "yeezy"
+  "body": {
```

Comment:
this is out of line with what the code implemented, also we should add every field here (headers, others?)

Reply:
+1

Reply:
also, can we define the proper output format for the user here, as well? we seem to be expecting from the user's function `{"status_code":1, "headers":{"yo":["dawg"]}, "body":"sup"}` but not explicitly defining it here. for HTTP we can just say `HTTP/1.1` easily enough but we need to define the json object we are expecting explicitly.